src/main/java/net/mrscauthd/beyond_earth/events/Methods.java

```diff
@@ -109,37 +109,25 @@ public static boolean spaceSuitCheck(LivingEntity entity) {
 
     public static boolean spaceSuitCheckBoth(LivingEntity entity) {
         boolean item3 = checkArmor(entity, 3, ModInit.OXYGEN_MASK.get());
-        boolean item2 = checkArmor(entity, 2, ModInit.SPACE_SUIT.get());
-        boolean item1 = checkArmor(entity, 1, ModInit.SPACE_PANTS.get());
-        boolean item0 = checkArmor(entity, 0, ModInit.SPACE_BOOTS.get());
-
         boolean item3_2 = checkArmor(entity, 3, ModInit.NETHERITE_OXYGEN_MASK.get());
+        if (!item3 && !item3_2) return false;
```

Comment:
If no oxygen_mask and no netherite_oxygen_mask, return false and skip checking all the other parts of the suit